grpc-gateway/search.pb.go

```diff
@@ -453,6 +453,55 @@ func (NestedQuery_ScoreMode) EnumDescriptor() ([]byte, []int) {
 	return file_yelp_nrtsearch_search_proto_rawDescGZIP(), []int{7, 0}
 }
 
+// Zero Terms options when analyzer removes all tokens.
+type MatchPhraseQuery_ZeroTerms int32
```

Comment:
Was this file regenerated? If not, would be good to do that.

Reply:
Yeah, these files were regenerated